src/Visualization/Python/ApplyPointwise.py

```diff
@@ -463,9 +530,45 @@ def shift_magnitude(
     task_id = progress.add_task("Applying to files")
     volfiles_progress = progress.track(volfiles, task_id=task_id)
     with progress:
-        apply_pointwise(volfiles_progress, kernels=kernels, **kwargs)
+        integrals = apply_pointwise(volfiles_progress,
+                                    kernels=kernels,
+                                    integrate=integrate,
+                                    **kwargs)
         progress.update(task_id, completed=len(volfiles))
 
+    # Write integrals to output file or print to terminal
+    if integrate:
+        integral_values = np.stack(list(integrals.values())).T
+        integral_names = list(integrals.keys())
+        if output:
+            if output.endswith(".txt"):
```

Comment:
or `.dat` or `.tsv`?

Reply:
I just removed this check. Now the output is an H5 file if the file extension is `.h5`, and a text file otherwise.